node/options.go

```diff
@@ -535,3 +535,10 @@ func WithAntifloodHandler(antifloodHandler P2PAntifloodHandler) Option {
 		return nil
 	}
 }
+
+// WithTxAccumulator sets up a transaction accumulator handler for the Node
+func WithTxAccumulator(accumulator Accumulator) Option {
+	return func(n *Node) error {
+		return n.setTxAccumulator(accumulator)
```

Comment:
check.ifNil set the txaccumulator than launch a method. To keep up the same style as for other woth options.

Reply:
ok, removed the set function from node.go